google-cloud-spanner-hibernate-testing/src/main/java/org/hibernate/userguide/model/Payment.java

```diff
@@ -26,7 +25,7 @@
     @GeneratedValue
     private Long id;
 
-    private BigDecimal amount;
+    private Long amount;
```

Comment:
Have you looked into using the new JDBC Driver and if it supports `BigDecimal`? Also, maybe there is a way to just extend the Simba one to deal with `BigDecimal`. Otherwise, we need to document somewhere that `BigDecimal` is not supported. Let's start a page in this repo keeping track of unsupported features like this one.

Reply:
Interestingly the new driver claims to support all of the JDBC types except NUMERIC and DECIMAL (which corresponds to BigDecimal). (you can see this in the file `JdbcParameterStore.java`

Reply:
If you want to try knut's driver, I built his shaded jar in some personal experiments here: https://github.com/dzou/knut-hibernate-experiment/tree/master/lib